releases/release-1.14/README.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes 1.14
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.14/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.14/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s114-minutes) (join [kubernetes-milestone-burndown@] to receive meeting invites)
+* Contact: [#sig-release] on slack, [kubernetes-sig-release@] on e-mail
+
+#### Tracking docs
+
+* TODO: [Enhancements Tracking Sheet](http://bit.ly/k8s114-enhancements)
+* TODO: [Bug Triage Tracking Sheet](http://bit.ly/k8s114-bugtriage)
+* TODO: [CI Signal Report](http://bit.ly/k8s114-cisignal)
+* [Retrospective Document](http://bit.ly/k8s114-retro)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/sig-release/ephemera/testing.md) (TODO: this guide should live elsewhere, ref: [kubernetes/sig-release#428](https://github.com/kubernetes/sig-release/issues/428))
+
+## tl;dr
+
+The 1.14 release cycle is proposed as follows:
+
+- **Monday, January 07**: Week 1 - Release cycle begins
+- **Tuesday, January 29**: Week 4 - [Enhancements Freeze]
+- **Thursday, March 07**: Week 9* - [Code Freeze]
+- **Monday, March 25**: Week 12 - Kubernetes v1.14.0 released
+
+
+## What Will We Do Differently This Release?
+
+* All proposed Enhancements for this release must have an associated KEP.
+  * There is no grandfathering; if the enhancement had no associated KEP before, it must have one now.
+  * The KEP must have a checklist of requirements necessary for a feature to land as alpha, beta, or stable.  That checklist must have at a minimum a test plan, and an upgrade/downgrade plan.
+* There will be no Code Slush.
+  * Previous release teams have said the only value of Code Slush was a clear one week warning that Code Freeze is coming
+  * Use the start of [Burndown] meetings as that date if you need it
+
+## Timeline
+
+
+| **What** | **Who** | **Jan** | **Feb** | **Mar** | **WEEK** | **CI SIGNAL** |
```

Comment:
The markdown is broken :stuck_out_tongue:

Reply:
Thanks, my local markdown editor hid that.

Reply:
I'll be honest, part of me wants to try re-inventing this wheel.  This table is not how I have things aligned in my head.  Sortof a question of time, and whether others are _really_ big fans of this format

Reply:
I'm curious to see other possible formats.  Currently it's overly dense and not super readable. 
And I'm pretty sure trending toward 0% of SIG leadership looks closely at this table, whether as the cycle is starting mapping out their prospective work for the quarter or later during the cycle to affirm current readiness relative to next process phase.

Reply:
Welp.  I didn't get much time to mess around and haven't found something I liked.  I reserve the right to change the formatting, but let's get the dates locked in